test/helpers/quictestlib.c

```diff
@@ -265,7 +265,7 @@ int qtest_create_quic_objects(OSSL_LIB_CTX *libctx, SSL_CTX *clientctx,
                 goto err;
         }
         /*
-         * TODO(QUIC SERVER):
+         * TODO(QUIC FUTURE):
          *    Currently the simplistic handler of the quic tserver cannot cope
          *    with noise introduced in the first packet received from the
          *    client. This needs to be removed once we have proper server side
```

Comment:
We *do* have proper server side handling - so this text no longer quite makes sense.

Reply:
yeah, thats reasonable, we should probably remove this alltogether, as we have issues to track the complete removal of tserver